src/Extensions/GridFieldDetailForm_ItemRequestExtension.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace DNADesign\Elemental\Extensions;
+
+use SilverStripe\ORM\FieldType\DBField;
+use SilverStripe\Core\Extension;
+use DNADesign\Elemental\Models\BaseElement;
+
+class GridFieldDetailForm_ItemRequestExtension extends Extension
```

Comment:
While the core SS API is set now and is stuck with underscores, we don't need to follow suit - can we use `GridFieldDetailFormItemRequestExtension` or `GridFieldDetailForm\ItemRequestExtension` instead?